README.md

```diff
@@ -19,6 +41,13 @@ npm install
 ```bash
 npm run serve
 ```
+- You can see the following words in the terminal if you success.
+```
+App running at:
+  - Local:   http://localhost:8080/
+  - Network: http://192.168.50.226:8080/
```

Comment:
The IP varies depending on your working environment. We may want to elaborate explicitly.